examples/compiled/rect_heatmap.vg.json

```diff
@@ -179,11 +179,6 @@
         },
         "axisY": {
             "minExtent": 30
-        },
```

Comment:
As mentioned on slack, I think the defaults are okay although I'm not a super big fan. 

The point is that I'm using the example to explain default color and thus it should not be changed.